cocktails9/cocktails9.xcodeproj/project.pbxproj

```diff
@@ -285,6 +286,7 @@
 				INFOPLIST_KEY_UIApplicationSceneManifest_Generation = YES;
 				INFOPLIST_KEY_UIApplicationSupportsIndirectInputEvents = YES;
 				INFOPLIST_KEY_UILaunchScreen_Generation = YES;
+				INFOPLIST_KEY_UILaunchStoryboardName = "Launch Screen.storyboard";
```

Comment:
You don't need Launch screen storyboard anymore if you have SwiftUI view as splash